### ISSUE TITLE
Some Australian Content Ratings not getting parsed correctly

### ISSUE THREAD

**Issue by purevertigo** [gboudreau/XBMCnfoTVImporter.bundle#109]
Australian Ratings -

MA15+
R18+
X18+

All get parsed without the trailing '+'

Can someone awesome please help fix?

thanks

**Comment by Squazz** [comment]
Is this still a problem?
Testing in my setup, it goes through just fine?
The regex for capturing the mpaa rating also states that it should capture the + signs
[A-z0-9-+/.]

**Comment by purevertigo** [comment]
Yes, this is still a problem. the + at the end does not get parsed through to Plex.
